qcodes/dataset/sqlite/queries.py

```diff
@@ -1590,3 +1590,16 @@ def remove_trigger(conn: ConnectionPlus, trigger_id: str) -> None:
         name: id of the trigger
     """
     transaction(conn, f"DROP TRIGGER IF EXISTS {trigger_id};")
+
+
+def get_db_location_from_conn(conn: ConnectionPlus) -> str:
```

Comment:
This is not needed. We already have path_to_dbfile

Reply:
does `path_to_dbfile` use the same query?

either way, it would be really nice to remove redundancy.